Autolink failed to get `react-native-code-push` deployment key

## Environment

System:
    OS: macOS 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i3-8100B CPU @ 3.60GHz
    Memory: 224.80 MB / 8.00 GB
    Shell: 5.7.1 - /usr/local/bin/zsh
  Binaries:
    Node: 12.4.0 - /usr/local/Cellar/node/12.4.0/bin/node
    Yarn: 1.16.0 - /usr/local/Cellar/node/12.4.0/bin/yarn
    npm: 6.9.0 - /usr/local/Cellar/node/12.4.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.0-rc.1 => 0.60.0-rc.1

## Description

I put `reactNativeCodePush_androidDeploymentKey` in `res/values/codepush.xml` with:

```xml
<resources>
    <string moduleConfig="true" name="reactNativeCodePush_androidDeploymentKey">example</string>
</resources>
```

But got error in `/android/build/generated/rn/src/main/java/com/facebook/react/PackageList.java`

```
error: cannot find symbol
      new AsyncStoragePackage(),new BlurViewPackage(),new RNBackgroundFetchPackage(),new CalendarEventsPackage(),new CodePush(getResources().getString(R.string.reactNativeCodePush_androidDeploymentKey), getApplicationContext(), BuildConfig.DEBUG),new RNDeviceInfo(),new ExtraDimensionsPackage(),new RNGestureHandlerPackage(),new Interactable(),new LinearGradientPackage(),new ReanimatedPackage(),new RNSharePackage(),new VectorIconsPackage(),new RNCWebViewPackage(),new RNFetchBlobPackage()
                                                                                                                                                               ^
  symbol:   variable reactNativeCodePush_androidDeploymentKey
  location: class string
```

## Reproducible Demo

https://github.com/robertying/autolink-issue-example

Run `cd android && ./gradlew assembleDebug` and you can get the error.

--- a/packages/platform-android/native_modules.gradle
+++ b/packages/platform-android/native_modules.gradle
@@ -145,7 +145,7 @@ class ReactNativeModules {
     String packageClassInstances = ""
 
     if (packages.size() > 0) {
-      packageImports = "import ${applicationId}.BuildConfig;\n\n"
+      packageImports = "import ${applicationId}.BuildConfig;\nimport ${applicationId}.R;\n\n"
       packageImports = packageImports + packages.collect {
         "// ${it.name}\n${it.packageImportPath}"
       }.join(';\n')

--- a/packages/platform-android/native_modules.gradle
+++ b/packages/platform-android/native_modules.gradle
@@ -148,7 +148,7 @@ class ReactNativeModules {
       packageImports = "import ${applicationId}.BuildConfig;\nimport ${applicationId}.R;\n\n"
       packageImports = packageImports + packages.collect {
         "// ${it.name}\n${it.packageImportPath}"
-      }.join(';\n')
+      }.join('\n')
       packageClassInstances = ",\n      " + packages.collect { it.packageInstance }.join(',')
     }
 

